components/03-modules/brief-info/_brief-info.scss

```diff
@@ -0,0 +1,74 @@
+$brief-info__padding                                       : $spacer--medium 0 !default;
+$brief-info__border-style                                  : 1px $gray-light solid !default;
```

Comment:
it is not only border style but whole border, therefore , should be `$brief-info__border-style`